finetune/download.py

```diff
@@ -10,26 +11,44 @@ def download_data_if_required():
     """ Pulls the pre-trained model weights from Github if required. """
     github_base_url = "https://raw.githubusercontent.com/IndicoDataSolutions/finetune/master/finetune/model/"
     s3_base_url = "https://s3.amazonaws.com/bendropbox/"
-
+    gpt2_base_url = "https://s3.amazonaws.com/bendropbox/gpt2/"
+    finetune_base_folder = os.path.dirname(finetune.__file__)
     file_list = [
-        (github_base_url, "encoder_bpe_40000.json"),
-        (github_base_url, "vocab_40000.bpe"),
-        (s3_base_url, "Base_model.jl"),
-        (s3_base_url, "SmallBaseModel.jl")
+        {
```

Comment:
Not sure how I feel about pulling all the base models whether you're going to use it or not? Its fine for now, but might not be as nice if/when we add more base models.